grpc/src/main/java/com/linecorp/armeria/internal/common/grpc/GrpcStatus.java

```diff
@@ -117,8 +119,8 @@ private static Status statusFromThrowable(Throwable t) {
             // instead.
             return s;
         }
-        if (t instanceof ClosedStreamException) {
-            return Status.CANCELLED;
+        if (t instanceof ClosedStreamException || t instanceof RequestTimeoutException) {
```

Comment:
Sorry, it is not related to the current changes in this PR. Initially, it tried to abort a call using `ctx.cancel()` instead `ServerCall.close()`. I found that a cause is not propagated with `ctx.cancel(cause)` version.

But, I think, it would be better to set a cause for `ClosedStreamException` and `RequestTimeoutException` for easier debugging. It violates the rule that `RequestTimeoutException` returns `DEADLINE_EXCEEDED`. Although it was a problem since it was specially handled by  https://github.com/line/armeria/blob/a953a36e96aabea6dcfb577518f9b26e3ba2f2e2/grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java#L314-L317